data/json/overmap/overmap_special/specials.json

```
@@ -911,6 +911,27 @@
     "occurrences": [ 0, 4 ],
     "flags": [ "CLASSIC", "WILDERNESS" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "Wind Turbine",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "wind_turbine_ground_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 2 ], "overmap": "wind_turbine_pillar_platform_north" },
+      { "point": [ 0, 0, 3 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 4 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 5 ], "overmap": "wind_turbine_pillar_platform_north" },
+      { "point": [ 0, 0, 6 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 7 ], "overmap": "wind_turbine_pillar_north" },
+      { "point": [ 0, 0, 8 ], "overmap": "wind_turbine_nacelle_north" },
+      { "point": [ 0, 0, 9 ], "overmap": "wind_turbine_roof_north" }
+    ],
+    "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "connection": "local_road", "from": [ 0, 0, 0 ] } ],
+    "locations": [ "land" ],
```

```suggestion
    "locations": [ "field" ],
```
Trees get in the way of adequate wind. Possibly also "special_field", although I don't know what that one is doing exactly. "lake_shore" might also be possible.